addon/helpers/class-names.js

```diff
@@ -2,5 +2,9 @@ import { helper } from '@ember/component/helper';
 import classnames from 'classnames';
 
 export default helper(function classNames(params, hash) {
```

Comment:
Perhaps we ought to rename the second param to `proxy` to more clearly communicate what it is